InternetHospitalWebUI/src/app/Services/registration.service.ts

```diff
@@ -18,7 +18,7 @@ export class RegistrationService {
     Email: new FormControl('', Validators.email),
     Password: new FormControl('', Validators.required),
     ConfirmPassword: new FormControl('', [Validators.required, compareValidator('Password')]),
-    Role: new FormControl('', Validators.required),
+    Role: new FormControl(''),
```

Comment:
Why role isn't required? If i'm hatsker, i can remove selected radio by devtools and send form without it.

Reply:
The role anyway is validated on back-end
Now we have the default checked patient radiobutton
so what the problem?